src/databricks/labs/ucx/install.py

```diff
@@ -339,6 +339,7 @@ def folder(self):
 
     def run(self):
         logger.info(f"Installing UCX v{PRODUCT_INFO.version()}")
+        self._enable_files_in_repos()
```

Comment:
Can we do it on the installer side (before we save() config)? Or even better - in blueprint, when we receive error that feature is disabled?.. do you remember the error code for that?

Reply:
This was the error:
 databricks.sdk.errors.mapping.NotFound: Files in Repos must be enabled for AUTO ExportFormat

Reply:
Moved to blueprint